html/index.html

```diff
@@ -0,0 +1,713 @@
+              </svg>
+              <h4>Tution and Fees</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#case"></use>
+              </svg>
+              <h4>Review & Rating</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+
+            <div class="started-block____right-box-item">
+              <svg width="24" height="24">
+                <use xlink:href="/images/sprite.svg#book"></use>
+              </svg>
+              <h4>Universoty Facilities</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+
+            <div class="started-block____right-box-item">
+              <svg width="18" height="18">
+                <use xlink:href="/images/sprite.svg#message"></use>
+              </svg>
+              <h4>Community Q&A</h4>
+            </div>
+            <div class="started-block____right-underline"></div>
+          </div>
+        </div>
+      </section>
+
+      <section class="blog-block">
+        <div class="blog-block____title">
+          <h2 class="block-title">Latest Blog</h2>
+          <div class="title-underline"></div>
+        </div>
+        <div class="blog-block____box">
+          <div class="blog-block____box-item">
+            <div class="blog-block____box-item-background"></div>
+            <p>DESIGN</p>
+            <img
+              class="blog-block____box-item-img"
+              width="328"
+              src="/images/blog-block/card-left.png"
+              alt="card-left"
+            />
+            <img
+              class="blog-block____box-item-man"
+              src="/images/courses-block/man_1.png"
+              alt="man_1"
+            />
+            <h3 class="block-subtitle">International Conference on Bussiness</h3>
+            <p>Read Now</p>
+          </div>
+          <div class="blog-block____box-item">
+            <div class="blog-block____box-item-background"></div>
+            <p>DESIGN</p>
+            <img
+              class="blog-block____box-item-img"
+              width="328"
+              src="/images/blog-block/card-left.png"
+              alt="card-left"
+            />
+            <img
+              class="blog-block____box-item-man"
+              src="/images/courses-block/man_1.png"
+              alt="man_1"
+            />
+            <h3 class="block-subtitle">International Conference on Bussiness</h3>
+            <p>Read Now</p>
+          </div>
+
+          <div class="blog-block____box-numbers">
+            <div class="block-subtitle">1</div>
+            <div class="block-subtitle">2</div>
+            <div class="block-subtitle">3</div>
+            <div class="block-subtitle">4</div>
+            <div class="block-subtitle">5</div>
```

Comment:
Don't use margin for alignment. Use flex alignments instead.

![image](https://user-images.githubusercontent.com/108632101/191867065-c6fd724e-bb4b-4616-9498-e259d94c0042.png)